packages/kbn-apm-synthtrace/src/lib/apm/apm_fields.ts

```diff
@@ -18,6 +18,13 @@ export type ApmApplicationMetricFields = Partial<{
   'jvm.memory.heap.used': number;
   'jvm.memory.non_heap.used': number;
   'jvm.thread.count': number;
+  'container.id': string;
```

Comment:
These are metadata fields, I think they belong somewhere else

Reply:
@dgieselaar where do you think they could fit better?  Would it better to create separate type for fields from different indices?

Reply:
You need to set these fields for the instance, not in the appMetrics call. I'm on my phone so can't check the other types, but pretty sure there is a better place (e.g. where container.id is located)

Reply:
Okay, thanks. container.id lives in instance